std/typecons.d

```diff
@@ -7143,3 +7143,265 @@ unittest
         ubyte, ubyte, T3, T3,
     );
 }
+
+/**
+ * Get the value of a field or property member. The member's name
+ * is resolved at runtime, rather than at compile-time.
+ *
+ * Note that currently only `pure @safe` property methods are supported.
```

Comment:
> `pure @safe` is the minimum restriction required to filter out the problematic stuff from `Object`

Note that the tests `rtFieldDispatch()` unit tests are `pure @safe nothrow @nogc`, even though the dispatch functions don't mandate the `nothrow @nogc` part.